### PR TITLE
Update Seraphim and Coalition maps

### LINKED ISSUE #356: Operation Kael: Easy difficulty bug
In the map editor one of the scripted attacks: P2KO1group4 has a misspelling. currently, for the easy difficulty, this  P2KO1group4_D` needs to be this P2KO1group4_D1.

### PR DESCRIPTION
**Pull request by Garanas**
Fixes #356

Includes two new coalition maps:

- Red Revenge
- Rebel's Rest

And one new Seraphim map:

- Overlord Surth Velsok

### CODE REVIEW

File: FAF_Coop_Operation_Ioz_Shavoh_Kael/UEFaiP3.lua [hunk 1]
```diff
@@ -4,20 +4,35 @@ local ScenarioFramework = import('/lua/ScenarioFramework.lua')
 local CustomFunctions = '/maps/FAF_Coop_Operation_Ioz_Shavoh_Kael/FAF_Coop_Operation_Ioz_Shavoh_Kael_CustomFunctions.lua'
 local ScenarioPlatoonAI = import('/lua/ScenarioPlatoonAI.lua')
 local ScenarioUtils = import('/lua/sim/ScenarioUtilities.lua')
+local CustomFunctions = import('/maps/FAF_Coop_Operation_Thisl_Eniz/FAF_Coop_Operation_Thisl_Eniz_CustomFunctions.lua')
```
Comment (speed2CZ) [inline FAF_Coop_Operation_Ioz_Shavoh_Kael/UEFaiP3.lua:7]: This line has to be deleted

File: FAF_Coop_Operation_Golden_Crystals/FAF_Coop_Operation_Golden_Crystals_scenario.lua [hunk 2]
```diff
@@ -9,14 +9,25 @@ ScenarioInfo = {
...
+    norushoffsetX_Player4 = 0.000000,
+    norushoffsetY_Player4 = 0.000000,
     Configurations = {
         ['standard'] = {
             teams = {
                 { name = 'FFA', armies = {'Player1','Civilians','QAI','Player2','Player3','Player4',} },
             },
             customprops = {
             },
-            factions = { {'uef', 'aeon', 'cybran'}, {'uef', 'aeon', 'cybran'}, {'uef', 'aeon', 'cybran'}, {'uef', 'aeon', 'cybran'} },
```
Comment (speed2CZ) [inline FAF_Coop_Operation_Golden_Crystals/FAF_Coop_Operation_Golden_Crystals_scenario.lua:20]: Not sure if this is intentional?

File: FAF_Coop_Operation_Rebel's_Rest/FAF_Coop_Operation_Rebel's_Rest_scenario.lua [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    norushoffsetX_Player4 = 0.000000,
+    norushoffsetY_Player4 = 0.000000,
+    Configurations = {
+        ['standard'] = {
+            teams = {
+                { name = 'FFA', armies = {'Player1','UEF1','UEF2','Player2','Player3','Player4',} },
+            },
+            customprops = {
+            },
+        },
```
Comment (speed2CZ) [inline FAF_Coop_Operation_Rebel's_Rest/FAF_Coop_Operation_Rebel's_Rest_scenario.lua:32]: missing available factions?

File: FAF_Coop_Operation_Red_Revenge/FAF_Coop_Operation_Red_Revenge_scenario.lua [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    norushoffsetX_Player4 = 0.000000,
+    norushoffsetY_Player4 = 0.000000,
+    Configurations = {
+        ['standard'] = {
+            teams = {
+                { name = 'FFA', armies = {'Player1','Order','Seraphim','QAI','UEF','Player2','Player3','Player4',} },
+            },
+            customprops = {
+            },
+        },
```
Comment (speed2CZ) [inline FAF_Coop_Operation_Red_Revenge/FAF_Coop_Operation_Red_Revenge_scenario.lua:36]: missing available factions?

**Review by speed2CZ — CHANGES_REQUESTED** [review]
- scenarios are missing `map_version` the server needs it to update the map correctly
- building from mobile factories has to be updated

**Review by speed2CZ — APPROVED** [review]
Looks good

**Comment by zqh9843** [comment]
Overlord Surth Velsok had a weird lighting bug the last time I played. But it's not consistent on every platform tho. Seems to affect AMD CPU+NVIDIA GPU combination the most.

**Comment by Garanas** [comment]
That can be tackled in a separate pull request.

**Pull request merged**